components/core/src/os/users/windows.rs

```diff
@@ -12,26 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::env;
 use std::path::PathBuf;
 
+use habitat_win_users::account::*;
```

Comment:
It's frowned upon to include an entire module in this way unless it's a module which just re-exports things for you or you're in a test module. This is totally fine for now, but just giving you a heads up!